cluster/saltbase/salt/docker/init.sls

```diff
@@ -195,9 +197,23 @@ net.ipv4.ip_forward:
 #    shasum <deb>
 # 6. Update override_deb, override_deb_sha1, override_docker_ver with new
 #    deb name, new hash and new version
+#    For example, for docker 1.8.3-0~wheezy_amd64.deb:
+#        docker_pkg_name - docker-engine
+#        override_deb - docker-engine_1.8.3-0~wheezy_amd64.deb
+#        override_deb_sha1 - f812e1134e7375b29886a372a6e24d04e1ee33c9
+#        override_docker_ver - 1.8.3-0~wheezy
+# 7. Sometimes debian package may not work, to use static linked binary version:
```

Comment:
Make sense.
I thought the binary package is another "official way" of docker deployment. I'll remove this and keep it in my note. :)